scout/server/blueprints/cases/templates/cases/cases.html

```diff
@@ -218,7 +218,9 @@ <h5 class="modal-title" id="sangerModalLabel">Sanger validations to evaluate:</h
     </td>
     <td>
       {{ case.analysis_date.date() }}
-      {% if case.is_rerun %}
+      {% if case.rerun_requested %}
+          <span class="badge badge-secondary pull-right">rerun pending</span>
```

Comment:
pull-right is deprecated in Bootstrap 4 and that's why the barge is aligned weirdly.

You could add **class="d-flex justify-content-between align-items-center"** to the td parent element instead